### PR TITLE
Add support for EnableEditorConfigSupport flag

### PR DESCRIPTION
**Pull request by Hoffs**
Similar to https://github.com/OmniSharp/omnisharp-vscode/pull/2836

Requires https://github.com/OmniSharp/omnisharp-roslyn/pull/1526

Added flag:
```
"omnisharp.enableEditorConfigSupport": true
```
which adds `FormattingOptions:EnableEditorConfigSupport=true` to omnisharp args

### CODE REVIEW

**Comment by akshita31** [comment]
@Hoffs  Could you please also add  a changelog entry that links to this PR and also the PR in omnisharp-roslyn

**Comment by akshita31** [comment]
@Hoffs Could you also add a "Contributed by" section as we usually call out community contributions explicitly on the change log.

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,7 +16,7 @@
 * Updates to Razor support
 * Made QuickInfo display more consistent with Visual Studio. ([#2610](https://github.com/OmniSharp/omnisharp-vscode/issues/2610))  (_Contributed by_ [@paladique](https://github.com/paladique))(PR: [#3090](https://github.com/OmniSharp/omnisharp-vscode/pull/3090/))
 * Added support for fading unnecessary code and using statements [#2873](https://github.com/OmniSharp/omnisharp-vscode/pull/2873)
-* Added a `omnisharp.enableEditorConfigSupport` setting to enable support for .editorconfig [#3136](https://github.com/OmniSharp/omnisharp-vscode/pull/3136) (PR: [omnisharp-roslyn#1526](https://github.com/OmniSharp/omnisharp-roslyn/pull/1526))
+* Added a `omnisharp.enableEditorConfigSupport` setting to enable support for .editorconfig [#3136](https://github.com/OmniSharp/omnisharp-vscode/pull/3136) (_Contributed by_ [@hoffs](https://github.com/hoffs))(PR: [omnisharp-roslyn#1526](https://github.com/OmniSharp/omnisharp-roslyn/pull/1526))
```
Comment (Hoffs) [inline CHANGELOG.md]: @akshita31 Added that as well.

Comment (akshita31) [reply CHANGELOG.md]: Please add @filipw for the omnisharp-roslyn PR and rebase. Thank you

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,7 +16,7 @@
 * Updates to Razor support
 * Made QuickInfo display more consistent with Visual Studio. ([#2610](https://github.com/OmniSharp/omnisharp-vscode/issues/2610))  (_Contributed by_ [@paladique](https://github.com/paladique))(PR: [#3090](https://github.com/OmniSharp/omnisharp-vscode/pull/3090/))
 * Added support for fading unnecessary code and using statements [#2873](https://github.com/OmniSharp/omnisharp-vscode/pull/2873)
-* Added a `omnisharp.enableEditorConfigSupport` setting to enable support for .editorconfig [#3136](https://github.com/OmniSharp/omnisharp-vscode/pull/3136) (_Contributed by_ [@hoffs](https://github.com/hoffs))(PR: [omnisharp-roslyn#1526](https://github.com/OmniSharp/omnisharp-roslyn/pull/1526))
+* Added a `omnisharp.enableEditorConfigSupport` setting to enable support for .editorconfig [#3136](https://github.com/OmniSharp/omnisharp-vscode/pull/3136) (_Contributed by_ [@hoffs](https://github.com/hoffs))(PR: [omnisharp-roslyn#1526](https://github.com/OmniSharp/omnisharp-roslyn/pull/1526) (_Contributed by_ [@filipw](https://github.com/filipw)))
```
Comment (Hoffs) [inline CHANGELOG.md]: @akshita31 Is this good enough?

Comment (akshita31) [reply CHANGELOG.md]: Yes thank you

**Pull request merged**